src/main/environments/environment.ts

```diff
@@ -1,16 +1,11 @@
-// This file can be replaced during build by using the `fileReplacements` array.
-// `ng build` replaces `environment.ts` with `environment.prod.ts`.
-// The list of file replacements can be found in `angular.json`.
-
 export const environment = {
   production: false,
+  fireBase: {
```

Comment:
Do not commit secret information in source control such as API key.

Reply:
removed